.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on: push
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [9.x, 10.x, 14.x, 16.x]
```

Comment:
12+ would be fine with me

Reply:
I leaved Node 10 in it for now if that's okay. We've a few services that need to support node 10. As it'll be a pain in the future, we can drop support for Node 10.

Reply:
Yeah that's fine :)